### PR TITLE
Update Inline elements table to fix #20, #22

### LINKED ISSUE #20: Mention the package tag in the style guide
@taroth21 rightfully remarked that we should mention that we are using the `<package/>` tag for RPMs (etc.).

### PR DESCRIPTION
**Pull request by dariavladykina**
Updates the "Inline elements" table in the "DocBook Tags" chapter to fix #20 and #22.

### CODE REVIEW

**Review by janajaeger — APPROVED** [review]
Perfect.

File: xml/docu_styleguide.taglist.xml [hunk 1]
```diff
@@ -926,7 +926,7 @@ link
        </screen>
       </entry>
       <entry>
-       A hypertext link
+       A hypertext link. Always use the secure protocol prefix (<literal>https://</literal>).
```
Comment (cwickert) [inline xml/docu_styleguide.taglist.xml:929]: "Always" is too strict as some servers/applications don't offer https.

```suggestion
       A hypertext link. Use the secure protocol prefix (<literal>https://</literal>), if possible.
```

**Review by cwickert — CHANGES_REQUESTED** [review]
I made two changes myself and have one suggestion. If you are fine with it, consider this approved (no need to re-review).

**Comment by dariavladykina** [comment]
Thanks for the additions @cwickert ! I've accepted the suggestion, too.

**Pull request merged**